sdk/attestation/azure-security-attestation/inc/ApiViewSettings.json

```
@@ -0,0 +1,16 @@
+{
+  "sourceFilesToProcess": [ "azure\\attestation.hpp" ],
+  "additionalIncludeDirectories": [
+    "../../../core/azure-core/inc"
+  ],
+  "sourceFilesToSkip": [
+  ],
+  "additionalCompilerSwitches": [],
+  "allowInternal": false,
+  "includeDetail": false,
+  "includePrivate": false,
+  "filterNamespace": "Azure::Security::Attestation",
+  "reviewName": "Azure Attestation",
```

Be consistent across all SDKs?

```suggestion
  "reviewName": "Azure Attestation API Review",
```